UI HOVER EFFECT/code/tilt.js

```diff
@@ -0,0 +1,183 @@
+                position: "absolute",
+                top: "0",
+                left: "0",
+                width: "100%",
+                height: "100%",
+                overflow: "hidden",
+                "pointer-events": "none"
+            }), Object.assign(this.glareElement.style, {
+                position: "absolute",
+                top: "50%",
+                left: "50%",
+                "pointer-events": "none",
+                "background-image": "linear-gradient(0deg, rgba(255,255,255,0) 0%, rgba(255,255,255,1) 100%)",
+                transform: "rotate(180deg) translate(-50%, -50%)",
+                "transform-origin": "0% 0%",
+                opacity: "0"
+            }), this.updateGlareSize())
+        }
+        updateGlareSize() {
+            if (this.glare) {
+                const t = 2 * (this.element.offsetWidth > this.element.offsetHeight ? this.element.offsetWidth : this.element.offsetHeight);
+                Object.assign(this.glareElement.style, {
+                    width: `${t}px`,
+                    height: `${t}px`
+                })
+            }
+        }
+        updateClientSize() {
+            this.clientWidth = window.innerWidth || document.documentElement.clientWidth || document.body.clientWidth, this.clientHeight = window.innerHeight || document.documentElement.clientHeight || document.body.clientHeight
+        }
+        onWindowResize() {
+            this.updateGlareSize(), this.updateClientSize()
+        }
+        setTransition() {
+            clearTimeout(this.transitionTimeout), this.element.style.transition = this.settings.speed + "ms " + this.settings.easing, this.glare && (this.glareElement.style.transition = `opacity ${this.settings.speed}ms ${this.settings.easing}`), this.transitionTimeout = setTimeout(() => {
+                this.element.style.transition = "", this.glare && (this.glareElement.style.transition = "")
+            }, this.settings.speed)
+        }
+        extendSettings(t) {
+            let e = {
+                    reverse: !1,
+                    max: 15,
+                    startX: 0,
+                    startY: 0,
+                    perspective: 1e3,
+                    easing: "cubic-bezier(.03,.98,.52,.99)",
+                    scale: 1,
+                    speed: 300,
+                    transition: !0,
+                    axis: null,
+                    glare: !1,
+                    "max-glare": 1,
+                    "glare-prerender": !1,
+                    "full-page-listening": !1,
+                    "mouse-event-element": null,
+                    reset: !0,
+                    gyroscope: !0,
+                    gyroscopeMinAngleX: -45,
+                    gyroscopeMaxAngleX: 45,
+                    gyroscopeMinAngleY: -45,
+                    gyroscopeMaxAngleY: 45,
+                    gyroscopeSamples: 10
+                },
+                i = {};
+            for (var s in e)
```

Comment:
Please prefer using {}. Even though not using them is syntactically correct, it usually makes the code difficult to read and understand